### ISSUE TITLE
Text overlapping in chrome latest version on using html2canvas javascript library

### ISSUE THREAD

**Issue by nishant-appvent** [niklasvh/html2canvas#3194]
![363686157-bb109689-7cf5-4068-bb86-a0770784d5f0](https://github.com/user-attachments/assets/088ab35e-c006-460f-9d88-fe431b02c4e4)
The actual correct HtmlElement

![363686266-6abffc22-5e4c-49bc-923d-1f46f0374d33](https://github.com/user-attachments/assets/c3d79967-9eff-4f55-ae98-ef4a7e24c057)
The image after converting it through html2canvas version "^1.4.1".

![KnvMmi9G](https://github.com/user-attachments/assets/ec9a704c-1422-45d9-ae4d-9afd76230ebd)
image on commenting this code in node_modules.
`var segmentWords = function (value, styles) {
        // if (FEATURES.SUPPORT_NATIVE_TEXT_SEGMENTATION) {
        //     // eslint-disable-next-line @typescript-eslint/no-explicit-any
        //     var segmenter = new Intl.Segmenter(void 0, {
        //         granularity: 'word'
        //     });
        //     // eslint-disable-next-line @typescript-eslint/no-explicit-any
        //     return Array.from(segmenter.segment(value)).map(function (segment) { return segment.segment; });
        // }
        return breakWords(value, styles);
    };`
### Bug reports:
I am facing this issue in latest chrome version i.e. 128.0.6613.84. While it was working fine with chrome version 127.0.6533.119. Can any one please suggest any solution for this?
Can you please look into it, @niklasvh?

### Specifications:
 * html2canvas version tested with: 1.4.1
 * Browser & version: Google chrome  128.0.6613.84.
 * Operating system: Ubuntu 20.04.6 LTS

**Comment by suddath-somani** [comment]
Facing the same issue. We are using jspdf to convert static html template to pdf, jsPdf internally uses html2canvas for this conversion. Everything was working fine before, but now after chrome v-128 update, the generated pdf seems to break.

![sample_jspdfpng](https://github.com/user-attachments/assets/34977c1b-c205-4927-88d8-87f29ccc2a81)

Can anyone suggest an alternative library to convert static html to pdf?

Note: The html to be downloaded is not being rendered, so the alternative should be able to handle the conversion without rendering.

**Comment by nishant-appvent** [comment]
We fixed this by removing "zoom" css property from that element. Have a look at this [chrome release note](https://developer.chrome.com/release-notes/128?hl=en#standardized_css_zoom_property), We were also facing this on using scale so we removed it.

**Comment by suddath-somani** [comment]
Yes i figured. Our application's layout is structured as such that it is not possible to remove zoom property at this point.
